app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/hplus/HPlusSupport.java

```diff
@@ -703,45 +703,44 @@ private void showText(String title, String body) {
         try {
             TransactionBuilder builder = performInitialized("notification");
 
-            byte[] msg = new byte[20];
-            for (int i = 0; i < msg.length; i++)
-                msg[i] = ' ';
+            String message = "";
 
-            msg[0] = HPlusConstants.CMD_ACTION_DISPLAY_TEXT;
+            if (title != null && title.length() > 0) {
 
-            String message = "";
+                if(LanguageUtils.transliterate()){
+                    title = LanguageUtils.transliterate(title);
```

Comment:
Affirmative, it works before your code, see to the end of PR500 (DeviceCommunicationService) and PR 513 (GBDeviceService). But you can manually call it without check (even it off in settings)